### PR TITLE
storing images into s3

### PR DESCRIPTION
**Pull request by SherrinRaji**
This branch includes:-
1. Storing the images for Plan and Feature into S3 bucket
2. Creating Private Storage for the above models on S3 bucket

### CODE REVIEW

File: core/admin.py [hunk 1]
```diff
@@ -4,7 +4,8 @@
 from django.utils.http import urlencode
 from django.utils.html import format_html
 from .models import Feature, License, LicensePlan, Plan, FeaturePlan, LicenseFeature
-
+from . import S3
+import requests
```
Comment (addu390) [inline core/admin.py:8]: Where is `requests` getting used?

File: licensing_platform/storage_backends.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from django.conf import settings
```
Comment (addu390) [inline licensing_platform/storage_backends.py:1]: Why is called `storage_backend`, it's `backend` implicit?

**Review by addu390 — COMMENTED** [review]
Can you share the IAM role (as per our discussion)?

File: core/admin.py (hunk 1 above)
Comment (SherrinRaji) [reply core/admin.py:8]: Oh my bad! I will remove that.

File: licensing_platform/storage_backends.py (hunk 2 above)
Comment (SherrinRaji) [reply licensing_platform/storage_backends.py:1]: Right! I will change it.

**Comment by SherrinRaji** [comment]
Hey @addu390,
I have made the changes requested in this PR from earlier. Please check and let me know if everything looks okay. Thanks! :)

**Review by addu390 — APPROVED** [review]
Alrighty, thank you for the changes, LGTM

**Pull request merged**